yt_dlp/extractor/motherless.py

```diff
@@ -1,32 +1,38 @@
 import datetime
 import re
+import urllib.parse
 
 from .common import InfoExtractor
-from ..compat import compat_urlparse
 from ..utils import (
     ExtractorError,
-    InAdvancePagedList,
-    orderedSet,
+    OnDemandPagedList,
     str_to_int,
     unified_strdate,
+    remove_end,
 )
 
 
 class MotherlessIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?motherless\.com/(?:g/[a-z0-9_]+/)?(?P<id>[A-Z0-9]+)'
+    _VALID_URL = r'https?://(?:www\.)?motherless\.com/(?:g/[a-z0-9_]+/|G[VIG]?[A-F0-9]+/)?(?P<id>[A-F0-9]+)'
     _TESTS = [{
-        'url': 'http://motherless.com/AC3FFE1',
-        'md5': '310f62e325a9fafe64f68c0bccb6e75f',
+        'url': 'http://motherless.com/EE97006',
         'info_dict': {
-            'id': 'AC3FFE1',
+            'id': 'EE97006',
             'ext': 'mp4',
-            'title': 'Fucked in the ass while playing PS3',
-            'categories': ['Gaming', 'anal', 'reluctant', 'rough', 'Wife'],
-            'upload_date': '20100913',
-            'uploader_id': 'famouslyfuckedup',
+            'title': 'Dogging blond Brit getting glazed (comp)',
+            'categories': ['UK', 'slag', 'whore', 'dogging', 'cunt', 'cumhound', 'big tits', 'Pearl Necklace'],
+            'upload_date': '20230519',
+            'uploader_id': 'deathbird',
             'thumbnail': r're:https?://.*\.jpg',
             'age_limit': 18,
-        }
+            'comment_count': int,
+            'view_count': int,
+            'like_count': int,
+        },
+        'params': {
+            # Downloads are currently broken because of an incomplete cert chain
+            'skip_download': True,
+        },
```

Comment:
pls explain

Reply:
I think it needs `'nocheckcertificate': True`

Reply:
That can be passed as `'params'`

Reply:
Ahh thanks, I didn't know about that.